inc/woocommerce-analytics/classes/wp-woocommerce-analytics-universal.php

```diff
@@ -0,0 +1,321 @@
+<?php
+/**
+ * Jetpack_WooCommerce_Analytics_Universal
+ *
+ * @package Jetpack
+ * @author Automattic
+ */
+
+/**
+ * Bail if accessed directly
+ */
+if ( ! defined( 'ABSPATH' ) ) {
```

Comment:
Just to be safe, maybe we should also add `class_exists( 'Jetpack_WooCommerce_Analytics_Universal' )` to this `if` block and exit out if it already does... that way if we do deploy this out to AT sites, and there is also a Jetpack release, we won't have duplicate logic.

Reply:
Nice... I ran into this problem locally

Reply:
I addressed this in https://github.com/Automattic/wc-calypso-bridge/pull/20/commits/4fd56b7008312750512405a2d4841d63eddba939

`class_exists( 'Jetpack_WooCommerce_Analytics' )` on the page where `Jetpack_WooCommerce_Analytics` is defined results in `true` even for a singular instance, so I've placed the check before any of the `wca` code is included.